### ISSUE TITLE
Support of customized models

### ISSUE THREAD

**Issue by DianCh** [warmspringwinds/pytorch-cpp#20]
Hi! Does pytorch-cpp support any conversion from customized models built in pytorch, or does it only support standard models (e.g. vgg, res-net) and your own segmentation models? 
Big thanks!

**Comment by walktree** [comment]
Yes, customized models are supported by libtorch. Actually, you could build any model since it has same interface as python version, e.g. I just migrated YOLO v3 in cpp api: https://github.com/walktree/libtorch-yolov3
